structuretoolkit/analyse/pymatgen.py

```diff
@@ -0,0 +1,75 @@
+from pymatgen.io.ase import AseAtomsAdaptor
+from pymatgen.analysis.local_env import VoronoiNN
+from pymatgen.core import Structure, Element
+import numpy as np
+import pandas as pd
+
+
+def get_stats(property_list, property_str):
+    """
+    Calculate statistical properties of a list of values.
+    Parameters:
+    property_list (list): A list of numerical values for which statistics are calculated.
+    property_str (str): A string prefix to be used in the resulting statistical property names.
+    Returns:
+    dict: A dictionary containing statistical properties with keys in the format:
+          "{property_str}_{statistic}" where statistic can be "std" (standard deviation),
+          "mean" (mean), "min" (minimum), and "max" (maximum).
+    Example:
+    >>> values = [1, 2, 3, 4, 5]
+    >>> get_stats(values, "example")
+    {'example_std': 1.4142135623730951,
+     'example_mean': 3.0,
+     'example_min': 1,
+     'example_max': 5}
+    """
+    return {
+        f"{property_str}_std": np.std(property_list),
+        f"{property_str}_mean": np.mean(property_list),
+        f"{property_str}_min": np.min(property_list),
+        f"{property_str}_max": np.max(property_list),
+    }
+
+
+def VoronoiSiteFeaturiser(structure, site):
```

Comment:
As this is a function, please use a lower case function name: 

```suggestion
def get_features_for_voronoi_site(structure, site):
```